crates/zksync/core/src/vm/inspect.rs

```diff
@@ -502,10 +502,18 @@ where
         .unwrap_or_default()
 }
 
-/// Maximum size allowed for factory_deps during create.
-/// We batch factory_deps till this upper limit if there are multiple deps.
-/// These batches are then deployed individually
-pub const MAX_FACTORY_DEPENDENCIES_SIZE_BYTES: usize = 100000; // 100kB
+lazy_static::lazy_static! {
```

Comment:
I'd use `once_cell` instead of `lazy_static` but this works anyways

Reply:
It shouldn't matter for this scenario, given that we plan on removing this.